libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnAsk.schema

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.ITriggerCondition)",
+    "title": "Ask Event",
+    "description": "This defines the actions to take when an OnAsk event is raised",
```

Comment:
Can you update the language around this to match the other Onxxx definitions?
    "title": "On ask",
    "description": "Actions to perform ask the user given a current state of memory???.",